code-challenges.test.js

```diff
@@ -33,11 +53,31 @@ const object2 = { number: 0 }
 // Expected output: "0 is divisible by three"
 const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
-
+describe("divisByThree", () => {
+    it("takes a object as an argument and decides if the number inside it is evenly divisible by three or not.", () => {
+        const object1 = { number: 15 }
+        const object2 = { number: 0 }
+        const object3 = { number: -7 }
+        expect(divisByThree(object1)).toEqual("15 is divisible by three")
+        expect(divisByThree(object2)).toEqual("0 is divisible by three")
+        expect(divisByThree(object3)).toEqual("-7 is not divisible by three")
+    })
+})
+//Fails at line 61 expect.
```

Comment:
Your test looks great. Since you have the test variables in the test (as they should be) they shouldn't be duplicated outside the test.